include/dronecore.h

```diff
@@ -34,6 +35,8 @@ class DroneCore
 
     ConnectionResult add_udp_connection();
     ConnectionResult add_udp_connection(int local_port_number);
+    ConnectionResult add_tcp_connection();
+    ConnectionResult add_tcp_connection(std::string ipAddress, int local_port_number);
```

Comment:
Here, we're missing the `add_serial_connection()`, right?

Reply:
i'll add that as part of #20 .

Reply:
@Rjasuja I don't understand. Shouldn't it be part of this PR?

Reply:
As there 2 issues raised for TCP & Serial connection support, @Rjasuja may be submitting different PR's for each. Both can be merged in the same PR.

Reply:
I only see this PR though 😄

Reply:
@julianoes I was in notion that Serial connection is another issue, all changes related can be submitted with the same. Do you want me to update here ?
I'll create PR for Serial connection support.

Reply:
Yes just update this PR until it's good to go in.